queries/synths/useBaseFeeRateQuery.ts

```diff
@@ -20,18 +19,14 @@ const useBaseFeeRateQuery = (
 		async () => {
 			const { SystemSettings } = synthetixjs!.contracts;
 
-			const [sourceCurrencyFeeRate, destinationCurrencyFeeRate] = (await Promise.all([
-				new Wei(
-					SystemSettings.exchangeFeeRate(
-						ethers.utils.formatBytes32String(sourceCurrencyKey as string)
-					)
+			const [sourceCurrencyFeeRate, destinationCurrencyFeeRate] = await Promise.all([
+				SystemSettings.exchangeFeeRate(
```

Comment:
`new Wei` function throws the `[big.js] invalid number` error because `SystemSettings.exchangeFeeRate` may return the value that it couldn't process.